### PR TITLE
<EditableSelector> and <Selector> - scss to stylable

### PR DESCRIPTION
**Pull request by roibWix**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6310

File: src/Selector/Selector.js [hunk 1]
```diff
@@ -1,10 +1,9 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames';
 import Checkbox from '../Checkbox';
 import RadioButton from '../RadioGroup/RadioButton/RadioButton';
 import Text from '../Text';
-import styles from './Selector.scss';
+import { style, classes } from './Selector.st.css';
```
Comment (EgoziE) [inline src/Selector/Selector.js:6]: We use the "st" alias for style.
Just for uniformity, please change it.

File: src/Selector/Selector.js [hunk 2]
```diff
@@ -32,6 +31,7 @@ class Selector extends React.PureComponent {
     showBelowNodeOnSelect: PropTypes.bool,
     belowNode: PropTypes.node,
     subtitleNode: PropTypes.node,
+    className: PropTypes.string,
```
Comment (EgoziE) [inline src/Selector/Selector.js:34]: Also please add it to the `d.ts` file

Comment (EgoziE) [reply src/Selector/Selector.js:34]: And type test (tsx file)

**Review by EgoziE — CHANGES_REQUESTED** [review]

File: src/Selector/Selector.js (hunk 1 above)
Comment (roibWix) [reply src/Selector/Selector.js:6]: done

File: src/Selector/Selector.js (hunk 2 above)
Comment (roibWix) [reply src/Selector/Selector.js:34]: what does it mean 'type test'?

Comment (roibWix) [reply src/Selector/Selector.js:34]: @EgoziE

**Review by EgoziE — APPROVED** [review]

**Pull request merged**